api/api.go

```diff
@@ -464,12 +476,25 @@ func SetupTLSConfig(tlsConfig *TLSConfig) (*tls.Config, error) {
 			return nil, err
 		}
 		tlsClientConfig.Certificates = []tls.Certificate{tlsCert}
+	} else if tlsConfig.CertFile != "" || tlsConfig.KeyFile != "" {
+		return nil, fmt.Errorf("both client cert and client key must be provided")
+	}
+
+	if len(tlsConfig.CertPEM) != 0 && len(tlsConfig.KeyPEM) != 0 {
```

Comment:
Right now `CertPEM` takes precedence over `CertFile`. I don't have a strong opinion, but I think it would better the other way around.

Reply:
Thanks! Fixed.